src/javafundamentals/collections/list/Ex03MaxValue.java

```diff
@@ -1,9 +1,24 @@
 package javafundamentals.collections.list;
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
 /**
  * Create a List with 20 random numbers.
  * Find the maximum value of the list and print it.
  */
 
 public class Ex03MaxValue {
+    public static void main(String[] args) {
+        List<Integer> randomNoList = new ArrayList<>();
+
+        for (int i = 0; i < 20; i++) {
+            randomNoList.add((int)(Math.random() * 100 -1) +1);
```

Comment:
You can use the class Random instead:
```
Random randomGenerator = new Random();
int randomNumber = randomGenerator.nextInt();
```

Reply:
Okay, I will try to remember to do that next time. Thanks